cloud/deploy/deploy.go

```diff
@@ -146,11 +146,11 @@ func removeDagsFromDockerIgnore(fullpath string) error {
 	return nil
 }
 
-func shouldIncludeMonitoringDag(deploymentType string) bool {
-	return !organization.IsOrgHosted() && !deployment.IsDeploymentDedicated(deploymentType) && !deployment.IsDeploymentHosted(deploymentType)
+func shouldIncludeMonitoringDag(deploymentType astroplatformcore.DeploymentType) bool {
+	return !organization.IsOrgHosted() && !deployment.IsDeploymentDedicated(deploymentType) && !deployment.IsDeploymentStandard(deploymentType)
```

Comment:
Did we rename the function IsDeploymentHosted to IsDeploymentStandard? I dont see that change in this PR

Reply:
IsDeploymentHosted was replaced by IsDeploymentStandard and IsDeploymentDedicated. IsDeploymentHosted is no longer used and the other two are used in different ways